src/messages/formatters.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * Wraps the content inside a codeblock with no language.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function codeBlock<C extends string>(content: C): `\`\`\`\n${C}\`\`\``;
+/**
```

Comment:
```suggestion
export function codeBlock<C extends string>(content: C): `\`\`\`\n${C}\`\`\``;

/**
```

For the love of god have mercy on my soul